languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -50663,11 +50663,11 @@ though, and has since been slightly modified)-->
             </rule>
             <rule>
                 <pattern>
-                    <token>opprobe</token>
+                    <token regexp="yes">opprobes?</token>
                 </pattern>
                 <message>Une erreur d'orthographe semble être détectée.</message>
-                <suggestion>opprobre</suggestion>
-                <example correction="opprobre">L'attitude inqualifiable du député jette l'<marker>opprobe</marker> sur toute la classe politique.</example>
+                <suggestion><match no="1" regexp_match="(opprob)(es?)" regexp_replace="$1r$2"/></suggestion>
+                <example correction="opprobre">L’attitude inqualifiable du député jette l'<marker>opprobe</marker> sur toute la classe politique.</example>
```

Comment:
no need for fancy apostrophes – regular apostrophes are transformed automatically in the message

Reply:
I'm experiencing difficulties to understand how those different style apostrophes are appearing ...